### PR TITLE
New sandbox branch

### PR DESCRIPTION
**Pull request by maria-robobug**
Creates a new up to date sandbox branch (sandbox2), based off master.

* I've tried to keep the structure and setup from the previous sandbox branch. So I've deleted anything that doesn't belong here, i.e Jenkinsfile, nginx rewrite conf etc.

* Upgraded to latest antora version, beta-3.

Next step will be to test this out with Netlify by pointing the sandbox site to this branch once this is merged.
Tested locally and everything works, so 🤞!

### CODE REVIEW

File: docsearch/docsearch-config.json.hbs [hunk 1]
```diff
@@ -1,113 +0,0 @@
-{
```
Comment (osfameron) [inline docsearch/docsearch-config.json.hbs:1]: Why are these .hbs files deleted? Ah, cos the whole docsearch is omitted?

File: README.adoc [hunk 2]
```diff
@@ -1,138 +0,0 @@
-= Couchbase Documentation Site
```
Comment (osfameron) [inline README.adoc:1]: unsure why this is deleted, won't that just cause merge issues if it gets updated in prod?

File: netlify/netlify.toml [hunk 3]
```diff
@@ -3,7 +3,9 @@ ignore = "false"
 publish = "public"
 
 [build.environment]
-ANTORA_CACHE_DIR = "~+/node_modules/.cache/antora"
+ANTORA_CACHE_DIR = "node_modules/.cache/antora"
```
Comment (osfameron) [inline netlify/netlify.toml:6]: > If the first path segment is a tilde directly followed by a plus sign (~+), or does not begin with an aforementioned prefix, the remaining path is resolved relative to the current working directory.
how does prod differ from sandbox here?

File: package.json [hunk 4]
```diff
@@ -4,9 +4,9 @@
   "homepage": "https://docs.couchbase.com",
   "private": true,
   "dependencies": {
-    "@antora/cli": "3.0.0-alpha.5",
-    "@antora/site-generator-default": "3.0.0-alpha.5",
-    "@antora/site-generator-ms": "git+https://gitlab.com/opendevise/oss/antora-site-generator-ms#v3.0.0-alpha.3",
+    "@antora/cli": "3.0.0-beta.3",
+    "@antora/site-generator-default": "3.0.0-beta.3",
```
Comment (osfameron) [inline package.json:8]: 🥳

File: antora-playbook.yml [hunk 5]
```diff
@@ -29,12 +29,12 @@ content:
     - url: .
       branches: HEAD
       start_path: home
-    # - url: https://git@github.com/couchbasecloud/couchbase-cloud
-    # branches: [AV-22069_Capella-GA-Docs]
-    # start_path: docs/public
-    # - url: https://git@github.com/couchbase/couchbase-operator
-    # branches: [2.3.0-beta, 2.2.x, 2.1.x, 2.0.x, 1.2.x, 1.1.x, 1.0.x]
-    # start_path: docs/user
+    - url: https://git@github.com/couchbasecloud/couchbase-cloud
```
Comment (osfameron) [inline antora-playbook.yml:32]: Looks like this cleanup is mostly enabling a whole lot of projects?
Sure that Capella and Mobile etc. are hot topics, but do we need all the other things (Talend etc.) below?
Just thinking about build times etc.

File: antora-playbook.yml [hunk 6]
```diff
@@ -60,7 +60,7 @@ content:
     - url: https://github.com/couchbase/docs-server
       branches: [release/7.0, release/6.6, release/6.5, release/6.0]
     - url: https://github.com/couchbase/docs-sdk-common
-      branches: [release/6.6, release/6.5, release/6.0, release/5.5]
+      branches: [release/7.0, release/6.6, release/6.5, release/6.0, release/5.5]
```
Comment (osfameron) [inline antora-playbook.yml:63]: can we remove the 5.5  release?

**Review by osfameron — APPROVED** [review]
Looks good!

File: docsearch/docsearch-config.json.hbs (hunk 1 above)
Comment (maria-robobug) [reply docsearch/docsearch-config.json.hbs:1]: Indeed, from what I can see in og sandbox and staging, the `docsearch` folder is not there.

File: README.adoc (hunk 2 above)
Comment (maria-robobug) [reply README.adoc:1]: Ah yeah, I got a bit excited copying old sandbox approach, this should probably stay 👍 .

File: antora-playbook.yml (hunk 5 above)
Comment (maria-robobug) [reply antora-playbook.yml:32]: This was available in og `sandbox` as well so I've not removed: https://github.com/couchbase/docs-site/blob/sandbox/antora-playbook.yml#L43

We can trim the fat later, but currently just trying to get this in a publishable state first.

File: antora-playbook.yml (hunk 6 above)
Comment (maria-robobug) [reply antora-playbook.yml:63]: Yeah I guess should be fine to remove 5.5 from common. @RichardSmedley, I don't remember if Ray merged his PR for removing 5.x branches, but should be fine to go no?

Comment (RichardSmedley) [reply antora-playbook.yml:63]: Yes, and anything else disappearing in the next 6 months or so - https://www.couchbase.com/support-policy/enterprise-software

Comment (RichardSmedley) [reply antora-playbook.yml:63]: cf https://github.com/couchbase/docs-site/commit/cf8453f52b4f66beef23f6e32585a732661b62dd#diff-0e8a7c2f91f241f59cc684e048c14593122447d39c4b8bfa6953cb5e91151aa2

Comment (RichardSmedley) [reply antora-playbook.yml:63]: btw https://github.com/couchbase/docs-site/pull/434 not yet merged, but no reason not to put saandbox ahead.  :+1:

**Pull request merged**